frontend/src/bundles/auth/auth-api.ts

```diff
@@ -35,6 +36,18 @@ class AuthApi extends HttpApiBase {
 
         return await response.json<UserSignUpResponseDto>();
     }
+
+    public async getByToken(): Promise<UserFindResponseDto> {
+        const response = await this.load(
+            this.getFullEndpoint(AuthApiPath.CURRENT_USER, {}),
+            {
+                method: 'GET',
+                contentType: ContentType.JSON,
+                hasAuth: true,
+            },
+        );
+        return await response.json<UserFindResponseDto>();
```

Comment:
You don't need return await (http://hassansin.github.io/Why-return-await-Is-a-Bad-Idea)